test/puppetlabs/puppetdb/integration/reports.clj

```diff
@@ -237,8 +237,8 @@
             logs  (get-href pdb (get-in report [:logs :href]))]
 
         (is (= #{{:name "total", :value 1, :category "events"}
-                 {:name "changed", :value 1, :category "resources"}
-                 {:name "total", :value 1, :category "changes"}
+                 {:name "changed", :value 0, :category "resources"}
+                 {:name "total", :value 0, :category "changes"}
```

Comment:
@robdaemon I'm about ready to merge this PR, but I can't quite figure out what caused the numbers in this file to change. Do you know/recall why this was necessary?

Reply:
I changed some of the fixtures, so that changed these values. I'd have to dig through it to find the specific ones